cmd/change_years.py

```diff
@@ -0,0 +1,12 @@
+import os
```

Comment:
@adfost this kind of script is not going to work we should use something like 

- https://github.com/palantir/go-license
- https://github.com/elastic/go-licenser

Reply:
no, we can run these tools to update the license on all files automatically, this way we need to only manage a single template for the license. 

Look at https://github.com/palantir/go-license for this project - it would be better to do that instead.  Lets take this PR I can send a change that handles this via CI/CD automatically.